EasyVCR.nuspec

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>EasyVCR</id>
+        <title>EasyVCR</title>
+        <version>0.1.0</version>
+        <authors>EasyVCR</authors>
+        <owners>EasyVCR</owners>
+        <projectUrl>https://github.com/EasyPost/easyvcr</projectUrl>
+        <license type="expression">MIT</license>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <description>A powerful library for recording and replaying HTTP requests</description>
+        <tags>EasyVCR HTTP requests record replay cassettes testing</tags>
+        <dependencies>
+            <group targetFramework="netstandard2.0">
+                <dependency id="Newtonsoft.Json" version="13.0.1" />
+            </group>
+            <group targetFramework="netcoreapp3.1">
+                <dependency id="Newtonsoft.Json" version="13.0.1" />
+            </group>
+            <group targetFramework="net5.0">
+                <dependency id="Newtonsoft.Json" version="13.0.1" />
+            </group>
+            <group targetFramework="net6.0">
+                <dependency id="Newtonsoft.Json" version="13.0.1" />
+            </group>
+        </dependencies>
+    </metadata>
+    <files>
+        <file src="lib\net\netstandard2.0\EasyVCR.dll" target="lib\netstandard2.0" />
+        <file src="lib\net\netstandard2.0\EasyVCR.XML" target="lib\netstandard2.0" />
+        <file src="lib\net\netcoreapp3.1\EasyVCR.dll" target="lib\netcoreapp3.1" />
+        <file src="lib\net\netcoreapp3.1\EasyVCR.XML" target="lib\netcoreapp3.1" />
+        <file src="lib\net\net5.0\EasyVCR.dll" target="lib\net5.0" />
+        <file src="lib\net\net5.0\EasyVCR.XML" target="lib\net5.0" />
+        <file src="lib\net\net6.0\EasyVCR.dll" target="lib\net6.0" />
+        <file src="lib\net\net6.0\EasyVCR.XML" target="lib\net6.0" />
+    </files>
+</package>
```

Comment:
Double check project for no end of file newlines. Your IDE should be configured to add these on save to avoid this moving forward.